qa/acceptance-tests/src/test/java/io/camunda/it/client/ActivateJobsTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.client;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.client.CamundaClient;
+import io.camunda.qa.util.multidb.MultiDbTest;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+
+@MultiDbTest
+public class ActivateJobsTest {
```

Comment:
💭 We already have [this test class](https://github.com/camunda/camunda/blob/955dd34fb9e82817d5599d3bcb5a7d939e05755d/zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/ActivateJobsTest.java#L65) for job activation tests. I think we don't need a multi-DB test for this as this is not DB-specific. 

The test in there for activation without jobs already works. Can we try to find out why that works but fails in production setups so we can hopefully create something reproducible that we can fix afterward? 😅

Reply:
I didn't notice this test yet. The test I wrote did fail first, so that does reproduce it correctly.

`@MultiDbTest` is a good point.

I'll check what the problem with the already existing test is.

Reply:
Fixed and extended the linked test, long polling was disabled.
Removed the unnecessary new test.